tools/c7n_mailer/README.md

```diff
@@ -257,6 +257,7 @@ and here is a description of the options:
 |:---------:|:----------------|:-----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | &#x2705;  | `queue_url`     | string           | the queue to listen to for messages                                                                                                                                                 |
 |           | `from_address`  | string           | default from address                                                                                                                                                                |
+|           | `endpoint_url`  | string           | The complete URL to use for the constructed client.                                                                                                                                                                |
```

Comment:
minor doc string here, probably should read - SQS API URL (for use with VPC Endpoints)